src/app/learn/get-started/basics/build-publish/index.html

```diff
@@ -0,0 +1,43 @@
+<stache
+  navOrder="7"
+  pageTitle="Build and Publish a SPA"
+  navTitle="Build and publish">
+
+  <stache-page-summary>
+    <p>
+      After you update the SKY UX CLI with the latest version, you can build and publish your SPA. 
+    </p>
+    <sky-alert alertType="info">
```

Comment:
`stache-internal` should be available if you wanted to add it to this PR.  https://github.com/blackbaud/stache2/blob/master/CHANGELOG.md#2170-2019-04-09

Reply:
I'm going to hold off on this one until tomorrow after a chat with Tim and Lindsey.

Reply:
I think we should probably hold off on this since the dev work is in flux. I think we should create a new issue to implement the restricted view component and directive throughout the SKY UX docs site after Tim finishes the dev work.

Reply:
👍 I like that idea and can create the issue for it.